packaging/python/README.md

```diff
@@ -1 +0,0 @@
-../../docs/install/python_wheels.md
```

Comment:
This is mostly to fix an issue with Podman on MacOS:

```plaintext
Error: invalid symlink "/var/tmp/libpod_builder742187951/build/README.md" -> "../../docs/install/python_wheels.md"
```

It also makes it more GitHub-friendly since GitHub doesn't follow symlinks, it just displays where it's pointing to.